libs/designsystem/shared/src/controls/label-helpers.ts

```diff
@@ -41,26 +41,17 @@ export function forwardAttributes(
   sourceElement: HTMLElement,
   attributes: string[],
   renderer: Renderer2,
-  targetElement?: HTMLElement
-): Record<string, any> {
-  const inheritedAttributes: Record<string, any> = {};
-
-  if (!sourceElement || !attributes || attributes.length === 0) {
-    return inheritedAttributes;
+  targetElement: HTMLElement
+) {
+  if (!sourceElement || !attributes || !renderer || !targetElement) {
+    return;
   }
 
   attributes.forEach((attribute) => {
     if (sourceElement.hasAttribute(attribute)) {
       const value = sourceElement.getAttribute(attribute);
-      inheritedAttributes[attribute] = value;
-
-      if (targetElement) {
-        renderer.setAttribute(targetElement, attribute, value);
-      }
-
+      renderer.setAttribute(targetElement, attribute, value);
```

Comment:
With this change, the docs for this function no longer matches. Is that intentional?

```
targetElement - (Optional) The HTML element to which the attributes 
should be applied. If not provided, the attributes will only be removed 
from the source element.
```